src/renderers/shared/fiber/__tests__/ReactIncrementalScheduling-test.js

```diff
@@ -449,373 +449,4 @@ describe('ReactIncrementalScheduling', () => {
     expect(ReactNoop.getChildren('b')).toEqual([span('b:3')]);
     expect(ReactNoop.getChildren('c')).toEqual([span('c:3')]);
   });
-
-  it('performs animation work in animation callback', () => {
-    class Foo extends React.Component {
-      componentDidMount() {
-        // Animation work that will get performed during animation callback
-        ReactNoop.performAnimationWork(() => {
```

Comment:
It seems to me that we still want a test that schedules animation work and deferred work in `componentDidMount`. These should *not* be synchronous since they've explicitly opted in to being async.